Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM python:3.9-slim
+FROM python:3.9-alpine
 WORKDIR /workdir
-COPY requirements.txt requirements.txt
-RUN pip install --no-cache-dir -r requirements.txt
-COPY src/*.py .
+COPY requirements.txt src/*.py ./
```

Comment:
This change impacts the developer experience since a line of change in the code needs to download all the dependencies again, thing that did not happen before